### ISSUE TITLE
[EPIC] Integrate Base chain on the wallet

### ISSUE THREAD

**Issue by shivekkhurana** [status-im/status-react#21526]
# Feature Issue

### User Story

As a user, 
I want to use Base chain on Status app 
**so that** I can interact with Base ecosystem

### Description

Currently, we have 3 chains hardcoded in our app: Mainnet, Opt and Arb. 
This was good enough when the designs were made initially, but now the ecosystem has accepted a multichain model. There are over 50 EVM chains. 

We cannot add all 50 in the current UX, but we can easily add one. In conversation with @VolodLytvynenko, we agreed to try and get Base working in 2.32. This is a "good-to-have" goal. In case it's not ready for 2.32, we'll move it to 2.33. 

The implementation detail is simple, we make no changes to the UX (everything stays the same), except we add one extra chain: Base.

Please check sub-tasks for actual implementation and acceptance criteria.

**Comment by churik** [comment]
@shivekkhurana 
I think  that is safe to say that we do not have capacity for this in 2.32.

Moving to 2.33

**Comment by briansztamfater** [comment]
@churik Agree, we also first need to remove multichain support (#21663) which is also scheduled for 2.33, otherwise we would need to add logic to the multichain implementation in order to support Base, which wouldn't make sense as the feature will be removed in favor of single-chain send / bridge.

**Comment by 86doteth** [comment]
looking forward to this. btw is there anything in the new design that makes it difficult/impossible to support any evm chain just like in v1? i get that there are many things to attend to in the next few releases, but would be great to have this ability again like before. discussed this before here https://github.com/status-im/status-mobile/issues/21062#issuecomment-2332487546 but maybe more is clear now? hopefully the simplified send and bridge gets us a step closer

no general evm support and no custom tokens (latter i [understood](https://discuss.status.app/t/feature-request-custom-token-lists/4839/2?u=86.eth) will be worked on indirectly) basically forces me and probably many others to export my seed phrase into some other wallet to get access to those other l2s and tokens. doesnt happen every day but basically just has to happen once and at that point my keycard setup is moot and might as well continue in the other app 😅

(not complaining to put this as top priority or anything, if we can get other cool stuff like say chat bots first 🤓(otherwise plz implement this today for my family))

**Comment by churik** [comment]
Thank you @86doteth for valuable insight, as simplified send flow is already on the way, may be @shivekkhurana we can consider to add custom chain only for sending assets? for 2.34 for example?
